visualization/include/pcl/visualization/pcl_visualizer.h

```diff
@@ -1708,6 +1708,9 @@ namespace pcl
                       const std::string &id = "ellipsoid",
                       int viewport = 0);
 
+        void
+        enableEDLRendering(int viewport = 0);
```

Comment:
A short doxygen documentation describing what EDL is and why it can be useful (and possibly that all rendered actors use EDL) would be nice